tests/unittests/Starboard/PathMapperTests.mm

```diff
@@ -69,6 +71,8 @@
 
     CPathMapper mapper16("/AppSupport/.\\?/");
     EXPECT_EQ_MSG(0, wcscmp(mapper16, L"d:\\temp\\AppSupport\\+"), "%S", (const wchar_t*)mapper16);
```

Comment:
(it means we don't have to rely on the logging infrastructure in the `_MSG` macro variants, and gtest can print expected/actual next to eachother)